script/upgrades/dependencies.json

```diff
@@ -1,5 +1,9 @@
 {
-  "42220": {},
+  "42220": {
+    "BridgedUSDC": "0xEB466342C4d449BC9f53A865D5Cb90586f405215",
+    "USDCUSDRateFeedAddr": "0xA1A8003936862E7a15092A91898D69fa8bCE290c",
+    "PartialReserveMultisig": "0x87647780180B8f55980C7D3fFeFe08a9B29e9aE1"
```

Comment:
how can I verify this address?

Reply:
https://docs.mento.org/mento/developers/deployments/addresses

and trust